### PR TITLE
Feature - additional availability

### PR DESCRIPTION
**Pull request by joefitter**
* Added migration to add project establishments table
* Updated projects list to include projects with establishments relations

### CODE REVIEW

File: schema/project-establishment.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  'removed'
+];
+
+class ProjectEstablishment extends BaseModel {
+  static get tableName() {
+    return 'projectEstablishments';
+  }
+
+  static get idColumn() {
+    return ['projectId', 'establishmentId'];
```
Comment (lennym) [inline schema/project-establishment.js:16]: Is this compatible with soft deleting things?

File: schema/project.js [hunk 2]
```diff
@@ -192,6 +228,19 @@ class Project extends BaseModel {
           to: 'establishments.id'
         }
       },
+      establishments: {
```
Comment (lennym) [inline schema/project.js:231]: Could we call this `additionalEstablishments`? It feels confusing to have `establishment` and `establishments` mean slightly different things.

File: schema/project-establishment.js (hunk 1 above)
Comment (joefitter) [reply schema/project-establishment.js:16]: we are hard deleting these

File: schema/project.js (hunk 2 above)
Comment (joefitter) [reply schema/project.js:231]: yup 👍

File: schema/project-establishment.js (hunk 1 above)
Comment (lennym) [reply schema/project-establishment.js:16]: Then we probably don't want the `deleted` column on the table.

Comment (joefitter) [reply schema/project-establishment.js:16]: correct!

**Pull request merged**